pkg/controller/machine.go

```diff
@@ -512,17 +512,23 @@ func (c *controller) machineDelete(machine *v1alpha1.Machine, driver driver.Driv
 			)
 			err = drainOptions.RunDrain()
 			if err != nil {
```

Comment:
why not `if err != nil && machine.Labels["force-deletion"] != "True" {`

Reply:
Makes more sense. Shall make this change.

Reply:
Made suggested changes - https://github.com/gardener/machine-controller-manager/pull/283/commits/52e1e875559ddd0aeb2c6c74f6dd1d931d54bd57